docs/LLS_Metallicities-Wotta/README.md

```diff
@@ -0,0 +1,52 @@
+## Create MCMC input files
```

Comment:
I believe you are referring to the "docs/LLS_Metallicities-Wotta" itself, wondering if it should be "docs/MCMC-Metallicities".  Is that correct?  Sounds good